### ISSUE TITLE
Search page: Play all videos in the playlist

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#1339]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
At the moment if u search for a playlist and u click on the playlist u will go to the playlist page. It would enhance user experience to also have an option to play all videos without going to the playlist page first. So there will be two options.

Option one: play all videos in playlist
Option two: view full playlist (This is already implemented)

**Describe the solution you'd like to see implemented**
I'm not sure it can be implemented this way, but hover over the playlist thumbnail to play all videos. See screen. 
If u can implement it like this u should create a button that says view full playlist (atm this is implemented when u click on the thumbnail).

**Describe alternatives you've considered**
None.

**Screenshot**
![Screenshot (206)](https://user-images.githubusercontent.com/73130443/119575113-0d486080-bda6-11eb-9f20-8f62616d095d.png)

**Additional context**
None.

**Comment by GilgusMaximus** [comment]
Good idea, but before anyone approaches this, I'd like to make some changes to how playlists are displayed beforehand. I am currently working on it for another feature, but it shouldn't be hard to translate it over to the normal search results.

**Comment by absidue** [comment]
This should be doable for the local API as YouTube gives us the first few videos in the playlist. To not change existing behaviour that users have got used to (clicking on the thumbnail to open the playlist page), I think the play all button should be added on top of the thumbnail, similarly to how videos have "Add/Remove from favourites" and "Open in external player" buttons on their thumbnails.

**Comment by jasonhenriquez** [comment]
Dependent on https://github.com/LuanRT/YouTube.js/issues/543 and https://github.com/iv-org/invidious/issues/4290
